qiskit/schemas/default_pulse_configuration_schema.json

```diff
@@ -140,13 +140,5 @@
             "type": "array"
         }
     },
-    "required": [
```

Comment:
Also a reminder to make sure that if some of the parameters end up being required, they should be kept in this section of the `.json`, in the hopes of facilitating keeping them in sync eventually.